### ISSUE TITLE
Max_results and page

### ISSUE THREAD

**Issue by dmdoniz** [alexmercerind/youtube-search-python#13]
Nice solution, but I‘ve got two issues (with v1.2.4)

1. Tried to change max_results to >20 , but always getting only 20 results.
2. I can only use page=1. With 2 or something else, I don‘t get a result - but there is more than one page availble when using the url in my browser

**Comment by alexmercerind** [comment]
Hi @dmdoniz !

This module is simply based on web-scraping, it makes following similar network request for getting results:
```
request = f"https://www.youtube.com/results?search_query={keyword}&page={offset}&sp={<search-filter>}"
```

- The "max_results" parameter is to limit the number of results you are receiving, not exceed the results that YouTube offers in that page..
  - I added it in because, assume a user wants only one video in the result, then the script will break the loop just after one video is sorted from the response and will give better performance to the user.

- We can make the number of results same as the user provided in "max_results", but for that we have to make multiple network requests with different "offset" values, which can affect performance. I haven't planned this, at the moment.

- Results are not always limited at 20, you can even get somewhere around 35 results too for some queries, it is over YouTube.

At the moment, for getting different videos in result, you can set offset > 1, to get videos on next pages of YouTube.
- "offset" is more like "page" but now I cannot change the name of this parameter as people are using this library now.


Thankyou!
Feel to free to re-open issue or provide feedback, in case you feel it.

**Comment by alexmercerind** [comment]
I very strongly recommend updating to v1.2.5 with:

```pip3 install youtube-search-python==1.2.5``` 

It has a big fix for a "mistake", I made in v1.2.4.
